service/storage_proxy.cc

```diff
@@ -433,8 +434,8 @@ class storage_proxy::remote {
         errors_info errors;
         ++p->get_stats().received_mutations;
         p->get_stats().forwarded_mutations += forward.size();
-        std::tuple<future<>, future<>> _ = co_await when_all(
-            coroutine::lambda([&] () -> future<> {
+        std::tuple<> _ = co_await coroutine::all(
```

Comment:
Is this return value valuable? We can just discard it with no?

Reply:
No, it can be discarded.